<?xml version="1.0"?>

<robot name="my_robot" xmlns:xacro="http://www.ros.org/wiki/xacro">
  
  
  <!-- <xacro:include filename="agv_car.xacro" /> -->
  <xacro:include filename="new_car.xacro" />

  <xacro:include filename="ros2_control_ackerman.xacro" />
  <!-- <xacro:include filename="ros2_control_bicycle.xacro" /> -->


  <!-- <xacro:include filename="lidar.xacro" />
  <xacro:include filename="depth_camera.xacro" /> -->

</robot>